src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs

```diff
@@ -33,13 +33,13 @@ public void Successor_ValidInput_ReturnsCorrectPostalCode(string postalCode, str
             Assert.AreEqual(postalCodeSuccessor, (new NLPostalCode(postalCode)).Successor.ToString());
         }
 
-        [Test]
-        [TestCase("0000 AA")]
-        [TestCase("0000")]
-        public void Predecessor_FirstInRange_ReturnsNull(string postalCode)
-        {
-            Assert.IsNull((new NLPostalCode(postalCode)).Predecessor);
-        }
+        //[Test]
+        //[TestCase("1000 AA")]
+        //[TestCase("1000")]
+        //public void Predecessor_FirstInRange_ReturnsNull(string postalCode)
```

Comment:
This test you can remove as I'm afraid there's no support for a min/max value other than zeroes and nines (an `A`s and `Z`s)